src/app.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
Prettier changed this to be lowercase instead: https://prettier.io/blog/2023/07/05/3.0.0.html#print-html5-doctype-in-lowercase-7391httpsgithubcomprettierprettierpull7391-by-fiskerhttpsgithubcomfisker